src/dotnet/APIView/APIViewWeb/LeanControllers/CommentsController.cs

```diff
@@ -169,6 +172,20 @@ public async Task<ActionResult> UpdateCommentTextAsync(string reviewId, string c
             return Ok();
         }
 
+        /// <summary>
+        /// Update comment severity
+        /// </summary>
+        /// <param name="reviewId"></param>
+        /// <param name="commentId"></param>
+        /// <param name="severity"></param>
+        /// <returns></returns>
+        [HttpPatch("{reviewId}/{commentId}/updateCommentSeverity", Name = "UpdateCommentSeverity")]
+        public async Task<ActionResult> UpdateCommentSeverityAsync(string reviewId, string commentId, [FromForm] CommentSeverity? severity)
```

Comment:
Assuming this is part of the internal API only, right?

Reply:
yes :)